data_dump/README.md

```diff
@@ -0,0 +1,152 @@
+realisation 0.) The dataset contains one dimension `name` for entries within a
+`GEN_KW` definition file, as well as the variables:
+
+* `standard_normal`: What ERT samples and is also the only thing that is stored
+  on-disk in the BlockFS system.
+* `transformed`: Values as transformed by ERT. This is what the user sees in
+  `parameters.txt`.
+* `probability_distribution`: The ERT name for the probability distribution used to convert `standard_normal` to `transformed`.
+* `probability_distribution_parameters`: JSON-ified dictionary of probability distribution parameters.
+
+## `GEN_DATA`
+
+All `GEN_DATA`s are in the same group under `/REAL_{index}/GEN_DATA`. This
+dataset contains the dimensions `report_step` as dates and `index` as the
+`GEN_DATA` indexing. The dates have been obtained by indexing the `time-map`
+vector at the given `REPORT_STEP` indices. Each NetCDF variable is a separate
+`GEN_DATA` entry.
+
+## `SUMMARY`
+
+All `SUMMARY`s are in the same group under `/REAL_{index}/SUMMARY`. This dataset
+contains the dimension `report_step`, which is the `time-map`. Each NetCDF
+variable is a separate `SUMMARY` entry.
+
+## `FIELD`
+
+Each ERT `FIELD` is a subgroup of `/REAL_{index}/FIELD`, where the name of the
+group is the name of the field. The dimensions are `x`, `y` and `z`, which
+correspond to the 3-D axes of fields. The data for the field is given by the
+`VALUES` variable.
+
+## `SURFACE`
+
+Each ERT `SURFACE` is a subgroup of `/REAL_{index}/SURFACE`, where the name of
+the group is the name of the surface. The data for the surface is given by the
+`VALUES` variable.
+
+# Reading
+
+To programmatically read data from the files it is recommended to use both the
+`netCDF4` package for low-level access, and `xarray` package for manipulating or
+converting the data.
+
+The `xarray` package does not make it easy to navigate groups which is why
+`netCDF4` is used. `xarray` uses `netCDF4` as a backend to open NetCDF version 4
+files, so both packages need to be installed anyway.
+
+## `netCDF4` package
+
+Groups can be navigated using `netCDF4`. First, open a file:
+
+``` python
+import netCDF4
+
+ds = netCDF4.Dataset("some-dataset.nc")
+```
+
+Groups are given using the `.groups` property. This returns a Python `dict` with
+the keys being the name of the group, and the value being a `netCDF4.Group`
+object. Groups may have subgroups.
+
+``` python
+for name, group in ds.groups.items():
+    for subname, subgroup in group.group.items():
+        print(f"{name}/{subname} = {subgroup}")
+```
+
+Specific subgroups can be accessed by `ds.groups["some_group"]` or
+`ds.group["some_group"]`. It is also possible to access subgroups by path, as if
+in a file structure: `ds["/foo/bar/baz"]`, which is equivalent to
+`ds.groups["foo"].groups["bar"].groups["baz"]`.
+
+NetCDF datasets have _variables_, which store actual data, and _dimensions_ (aka
+_coordinates_), which are the axes for the data. Within a group, if two
```

Comment:
I read this as if coordinates and dimensions are the same thing, but are they not slightly different?
From my understanding, dimensions define the structure and shape of the data, whereas coordinates provide a way to label the position along each dimension with meaningful names.
Perhaps just drop the "a.k.a coordinates"?

Reply:
Good point!